MiniApp/Classes/Display/MimeTypes.swift

```diff
@@ -0,0 +1,74 @@
+internal let defaultMimeType = "text/html"
+
+internal let mimeTypes = [
+    "html": "text/html",
+    "htm": "text/html",
+    "css": "text/css",
+    "xml": "text/xml",
+    "gif": "image/gif",
+    "jpeg": "image/jpeg",
+    "jpg": "image/jpeg",
+    "js": "application/javascript",
+    "rss": "application/rss+xml",
+    "txt": "text/plain",
+    "png": "image/png",
+    "tif": "image/tiff",
+    "tiff": "image/tiff",
+    "ico": "image/x-icon",
+    "jng": "image/x-jng",
+    "bmp": "image/x-ms-bmp",
+    "svg": "image/svg+xml",
+    "svgz": "image/svg+xml",
+    "webp": "image/webp",
+    "woff": "application/font-woff",
+    "json": "application/json",
+    "doc": "application/msword",
+    "pdf": "application/pdf",
+    "rtf": "application/rtf",
+    "m3u8": "application/vnd.apple.mpegurl",
+    "xls": "application/vnd.ms-excel",
+    "eot": "application/vnd.ms-fontobject",
+    "ppt": "application/vnd.ms-powerpoint",
+    "der": "application/x-x509-ca-cert",
+    "pem": "application/x-x509-ca-cert",
+    "crt": "application/x-x509-ca-cert",
+    "xhtml": "application/xhtml+xml",
+    "docx": "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
```

Comment:
Why add all of these extra mime types? Are there issues with other file types other than SVG?

I just get concerned when there is extra code which isn't needed. It could have unintended consequences (and it's more code to maintain). This should be updated to match only the mime types that the backend supports:

```
  'text/html',
  'application/javascript',
  'application/json',
  'text/css',
  'image/png',
  'image/jpeg',
  'image/gif',
  'image/svg+xml',
  'audio/mpeg',
  'audio/mp3'
```

Reply:
Got it 👍

Reply:
Done

Reply:
Nice, thanks!